dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -160,6 +160,7 @@ def get_torchbench_tpu_config(
     runtime_version: resource.RuntimeVersion,
     time_out_in_min: int,
     use_xla2: bool = False,
+    reserved: bool = True,
```

Comment:
What does "reserved" mean?

Reply:
This is for specifying whether the TPU resource is a reserved resource or not.  New TPU resources in `cloud-ml-benchmarking` are all unreserved.